[2022-12-08 16:26:24,252] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:26:20.043649+00:00 [queued]>
[2022-12-08 16:26:24,258] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:26:20.043649+00:00 [queued]>
[2022-12-08 16:26:24,258] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:26:24,258] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-08 16:26:24,258] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:26:24,268] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): move_data_from_redshift_to_s3> on 2022-12-08 16:26:20.043649+00:00
[2022-12-08 16:26:24,273] {standard_task_runner.py:52} INFO - Started process 621 to run task
[2022-12-08 16:26:24,275] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'move_data_from_s3_to_redshidt_with_awswrangler', 'move_data_from_redshift_to_s3', 'manual__2022-12-08T16:26:20.043649+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/move_data_s3redshift.py', '--cfg-path', '/tmp/tmptoc09p_d', '--error-file', '/tmp/tmp2fa26yo7']
[2022-12-08 16:26:24,276] {standard_task_runner.py:80} INFO - Job 100: Subtask move_data_from_redshift_to_s3
[2022-12-08 16:26:24,311] {task_command.py:369} INFO - Running <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_redshift_to_s3 manual__2022-12-08T16:26:20.043649+00:00 [running]> on host 8458a2c1ba2e
[2022-12-08 16:26:24,365] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=move_data_from_s3_to_redshidt_with_awswrangler
AIRFLOW_CTX_TASK_ID=move_data_from_redshift_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T16:26:20.043649+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-08T16:26:20.043649+00:00
[2022-12-08 16:26:24,378] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:26:24,764] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:26:25,770] {python.py:173} INFO - Done. Returned value was: None
[2022-12-08 16:26:25,780] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=move_data_from_s3_to_redshidt_with_awswrangler, task_id=move_data_from_redshift_to_s3, execution_date=20221208T162620, start_date=20221208T162624, end_date=20221208T162625
[2022-12-08 16:26:25,809] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-08 16:26:25,835] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
